.vscode/launch.json

```diff
@@ -0,0 +1,35 @@
+{
+  // Use IntelliSense to learn about possible attributes.
```

Comment:
I'm not familiar with the VSC config but if you want to include a comment, it would be more appropriate to use JS instead of JSON because JSON doesn't allow comments (excluding comments defined in a custom way like being key/values with some special key prefix).